mlir/tools/mlir-miopen-lib/mlir-miopen-lib-test.cpp

```diff
@@ -43,6 +43,8 @@ int main(int argc, char **argv) {
     std::cout << "ExecutionDims - globalSize=" << globalSize
               << ", localSize=" << localSize << std::endl;
 
+  } else if (option.getValue() == "workspace") {
+    std::cout << "Workspace=" << miirGetWorkspaceSize(handle) << std::endl;
```

Comment:
Mind adding at least 1 unit test on the library test side with workspace size no equal to zero?

Reply:
We do have a test to check non-zero workspace size. Please refer to `workspace.mlir` in this PR.